### ISSUE TITLE
Open screenshot in MSPaint in Windows 11

### ISSUE THREAD

**Issue by RensDuijsens** [ShareX/ShareX#6488]
In Windows 10 it was very easy to make ShareX automatically open MSPaint for further editing after taking the screenshot.
In '**Task settings / Actions**' enter the path "_C:\Windows\System32\mspaint.exe_" and in 'Arguments' enter "_$input_".
 
In Windows 11 the location "_C:\Windows\System32\mspaint.exe_" is no longer valid, so that does not work anymore.

It is easy to find the new location however.
Open an explorer. Type "_mspaint_" in the path-input-box and press enter.
MSPaint will open.
Now go back to the explorer and click the path-input-box again. It will show the full path to MSPaint.
In my case it is: "_C:\Program Files\WindowsApps\Microsoft.Paint_11.2206.6.0_x64__8wekyb3d8bbwe\PaintApp\mspaint.exe_"
ShareX will automatically translate that to "_**%ProgramFiles%\WindowsApps\Microsoft.Paint_11.2206.6.0_x64__8wekyb3d8bbwe\PaintApp\mspaint.exe**_"

When I test this full path in the 'Run' box, MSPaint will open just fine.
However.....when I enter this path in ShareX....nothing happens.

How do I make MSPaint open afther the screenshot in Windows 11 please?

**Comment by welpdx** [comment]
Yoooo it took my a 3 hr rabbit hole until I finally figured it out!

### finding mspaint path

So I tried what you did but instead of in WindowsApps, I just did `where mspaint` in cmd and I eventually figured out that this path is better for me: `%localappdata%\Microsoft\WindowsApps\mspaint.exe`. 

I then put this in Actions like you did, I also created a Task that "Perform Action" after Capture, but it didn't launch mspaint, just like you.

### Rabbit Hole

Here's the rabbit hole: I then thought it was a Admin Priv thing because I couldn't copy that mspaint to sys32. I tried downloading a win 7 version but that couldn't open in sys 32. I then downloaded a winxp version of mspaint but the Task in sharex still didnt open the mspaint to edit the capture. 

### Then I figured it out!!!

According [this blessed post made 7 years ago](https://www.reddit.com/r/sharex/comments/46gp39/comment/d1rxukw/?utm_source=share&utm_medium=web2x&context=3), I eventually figured out that you have to add "Save image to file" first before "Perform Actions"!!!! 
\*sigh\*

### tl;dr
In order to make this work, it is **necessary** to have "Save image to file" then "Perform Actions".  

![img](https://i.imgur.com/W8ira4y.png)
